src/routes/tutorials/index.svelte

```diff
@@ -69,14 +69,5 @@
       <span slot="title">Earn PNG</span>
       <span slot="description">Learn what PGL tokens are and how to use them to earn PNG</span>
     </TutorialCard>
-
-    <TutorialCard
-      href="/tutorials/claim-png"
-      class="bg-gradient-to-br from-yellow-400 to-orange-500"
-      texture="purty-wood"
```

Comment:
Let's delete `static/textures/purty-wood.png`.

Reply:
should we leave it incase we need another tutorial in the future?

Reply:
I'd say no, we can easily add it back if necessary as it's in the git history.

https://www.transparenttextures.com/purty-wood.html

Reply:
Ideally we won't need tutorials at all once our UI/UX is good enough :)